src/sparseml/yolov8/utils/helpers.py

```diff
@@ -69,3 +71,14 @@ def create_grad_sampler(
         / train_loader.batch_size,
     )
     return grad_sampler
+
+
+def detach(x: Union[torch.Tensor, List, Tuple]):
```

Comment:
Would it be better to move this func into sparseml/pytorch/utils/helpers.py so it could be reused more?

Reply:
Yes, definitely. If we believe it would reused, that's a better place.